framework/application/application.h

```diff
@@ -84,6 +84,8 @@ class Application final
 
     void StopRunning() { running_ = false; }
 
+    const decode::FileProcessor* GetFileProcessor() const { return file_processor_; }
```

Comment:
I think it would be better if this returned directly the frame number instead of the file processor

Reply:
Done !